apps/estuary/mix.lock

```diff
@@ -1,12 +0,0 @@
-%{
```

Comment:
Why was this guy deleted?

Reply:
This is not required for each app. 
-Sukaant

Reply:
Ok, I missed that. Thanks.